xapi-datamodel.opam

```diff
@@ -17,7 +17,6 @@ depends: [
   "rpclib"
   "base-threads"
   "xapi-consts"
-  "xapi-schema"
```

Comment:
This probably was needed before this reversal: https://github.com/xapi-project/xen-api/commit/9608b74224f59c8e6c2b59a039a40dc609618a1c